In [1]:
import numpy as np

In [2]:
divs = 52.53

In [16]:
for price in range(290, 460, 30):
    percent = round((divs * 100) / price, 2)
    print(price, ' ' ,percent)

290   18.11
320   16.42
350   15.01
380   13.82
410   12.81
440   11.94


Рассчитал процент дивов (52.53р) от цены 

цена  %
300   17.51
301   17.45
302   17.39
303   17.34
304   17.28
305   17.22
306   17.17
307   17.11 - размер дивов здесь покрывает текущую инфляцию 17,1%
308   17.06
309   17.0
310   16.95
311   16.89
312   16.84
313   16.78
314   16.73
315   16.68
316   16.62
317   16.57
318   16.52
319   16.47
320   16.42

Текущая инфляция за май 17,1% (см www.cbr.ru), а это у нас цена 307, учтывая это, не думаю, что сильно ниже 307 просядет. Хотя конечно кто знает. 

In [5]:
for price in np.arange(290, 295, 0.1):
    percent = round((divs * 100) / price, 2)
    print(round(price, 1), ' ' ,percent)    

290.0   18.11
290.1   18.11
290.2   18.1
290.3   18.1
290.4   18.09
290.5   18.08
290.6   18.08
290.7   18.07
290.8   18.06
290.9   18.06
291.0   18.05
291.1   18.05
291.2   18.04
291.3   18.03
291.4   18.03
291.5   18.02
291.6   18.01
291.7   18.01
291.8   18.0
291.9   18.0
292.0   17.99
292.1   17.98
292.2   17.98
292.3   17.97
292.4   17.97
292.5   17.96
292.6   17.95
292.7   17.95
292.8   17.94
292.9   17.93
293.0   17.93
293.1   17.92
293.2   17.92
293.3   17.91
293.4   17.9
293.5   17.9
293.6   17.89
293.7   17.89
293.8   17.88
293.9   17.87
294.0   17.87
294.1   17.86
294.2   17.86
294.3   17.85
294.4   17.84
294.5   17.84
294.6   17.83
294.7   17.82
294.8   17.82
294.9   17.81
